scheduler.py

```diff
@@ -63,7 +63,7 @@
 
 for i in range(num_shifts):
     for j in range(num_employees):
-        dqm.set_linear_case(j, i, dqm.get_linear_case(j, i) - gamma*(2*num_per_shift+1))
+        dqm.set_linear_case(j, i, dqm.get_linear_case(j, i) - gamma*(2*num_per_shift-1))
```

Comment:
Not sure if I'm missing something here, but why do we need to get_linear_case(j, i) for this? Wouldn't that always be 0? (same for below with get_quadratic_case)

Reply:
Good catch! somehow the preferences piece got cut out and employee preferences were not being used in the DQM.